example/configmap-rsyslog-auditd.yaml

```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+---
+kind: ConfigMap
```

Comment:
We could also adapt the example Shoot spec to contain reference to this ConfigMap. The same way as for the Secret: https://github.com/gardener/gardener-extension-shoot-rsyslog-relp/blob/71f15814a648731897101c2667b352efbcaca27b/example/shoot.yaml#L41-L46

For testing it would be much easier to uncomment instead of building the needed partial spec again.